package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@anchore/sbom-action",
+  "version": "0.1.0",
+  "private": true,
+  "description": "GitHub Action for creating software bill of materials",
+  "main": "src/runSyftAction.ts",
+  "scripts": {
+    "build": "tsc --noEmit",
+    "format": "prettier --write **/*.ts",
+    "format-check": "prettier --check **/*.ts",
+    "lint": "eslint src/**/*.ts",
+    "package": "ncc build src/runSyftAction.ts -m -o dist/runSyftAction --source-map --license licenses.txt && ncc build src/attachReleaseArtifacts.ts -m -o dist/attachReleaseArtifacts --source-map --license licenses.txt && ncc build src/downloadSyft.ts -m -o dist/downloadSyft --source-map --license licenses.txt",
+    "test": "jest",
+    "all": "npm run build && npm run format && npm run lint && npm run package && npm test",
+    "prepare": "husky install",
+    "prettier": "prettier -w src",
+    "precommit": "pretty-quick --staged && npm run package && git add dist/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/anchore/sbom-action.git"
+  },
+  "keywords": [
+    "actions",
+    "node",
+    "setup"
+  ],
+  "author": "",
```

Comment:
Do you want to include yourself as the author?

Reply:
I actually don't know the right thing to do here... did for now